shoplist/src/main/java/com/netguru/codereview/network/model/ShopListResponse.kt

```diff
@@ -0,0 +1,10 @@
+package com.netguru.codereview.network.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+data class ShopListResponse(
+    val list_id: String,
```

Comment:
Keep code style, refactor data model properties to camel case, use
com.google.gson.annotations.SerializedName

Convert list_id to listId
Check whether the list_id/listId type is a String?
I think it should be Int